DWAPlannerROS:

  max_vel_x: -2.5 
  min_vel_x: -0.5
  max_vel_theta: 1.0 
  min_in_place_vel_theta: 0.1

  acc_lim_theta: 3.2
  acc_lim_x: 2.5
  acc_lim_y: 2.5

  escape_vel: -0.3

  holonomic_robot: false
